src/platforms/platform.dom.js

```diff
@@ -304,17 +304,12 @@ function removeResizeListener(node) {
 	}
 }
 
-function injectCSS(platform, css) {
+function injectCSS(css) {
 	// https://stackoverflow.com/q/3922139
-	var style = platform._style || document.createElement('style');
-	if (!platform._style) {
```

Comment:
`injectCSS()` is only called once so we don't have to care about caching created element.

Reply:
The reason this cached is for the case of multiple charts on a single page. We only want 1 style element the

Reply:
But the styles should be injected multiple times? Please note that `style.appendChild(document.createTextNode(css));` wasn't part of that `if`.

Reply:
No, the style in `platform.initialize()` shouldn't be injected multiple times, but `injectCSS()` could be called multiple time to inject different style under the same `style` element. That's not the case currently but that could change so `injectCSS()` still need to be able to be called multiple times (which I guess is still the case).

Reply:
It's still possible to call `injectCSS()` multiple times but that will cause multiple link elements to be added. I didn't benchmark but I guess this is better from a performance perspective cause replacing may cause a more expensive rerendering.